JSDemos/Demos/DataGrid/GridColumnsHidingPriority/description.md

```diff
@@ -1 +1,3 @@
-The **DataGrid** can hide columns according to&nbsp;the specified priority if&nbsp;screen width is&nbsp;not enough to&nbsp;display all of&nbsp;them. The hiding priority is&nbsp;specified using the **hidingPriority** field of&nbsp;an&nbsp;item in&nbsp;the **columns** array. Switch between the landscape and portrait layouts to&nbsp;see how the **DataGrid** looks on&nbsp;wide and narrow screens.
\ No newline at end of file
+**DataGrid** automatically adapts its layout to screens of different widths. If screen width is not enough to display all columns, the widget hides them one by one. Set the [columnHidingEnabled](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#columnHidingEnabled) option to **true** to enable this feature.
```

Comment:
The reader already read that in the previous demo description. Let's try start with

"When the DataGrid adapts its layout to small screen width, it hides columns based on [hidingPriority]()."

Then continue describing how this works. Finally, deescribe what's happening in the demo.